examples/models/file/latex_extension_with_mathjax.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from scipy.special import jv
+
+from bokeh.models import Label
+from bokeh.palettes import Spectral4
+from bokeh.core.properties import (
+  Nullable,
+  Float
+)
+from bokeh.plotting import figure, output_file, show
+from bokeh.util.compiler import TypeScript
+
+output_file('latex_extension_with_mathjax.html')
+
+class LatexLabel(Label):
+    width = Nullable(Float, help="""
+    The gradient of the line, in data units
+    """)
+    height = Nullable(Float, help="""
+    The gradient of the line, in data units
+    """)
+    """A subclass of `Label` with all of the same class attributes except
+    canvas mode isn't supported and DOM manipulation happens in the TypeScript
+    superclass implementation that requires setting `render_mode='css'`).
+
+    Only the render method of LabelView is overwritten to perform the
+    text -> latex (via katex) conversion
```

Comment:
Shouldn't this be "Mathjax" here?
```suggestion
    text -> latex (via MathJax) conversion
```